CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 ### New
 - Can use Pod Identity with Azure Event Hub scaler ([#994](https://github.com/kedacore/keda/issues/994))
 - Introducing InfluxDB scaler ([#1239](https://github.com/kedacore/keda/issues/1239))
+- Introducing Mongo scaler ([#1465](https://github.com/kedacore/keda/issues/1465))
```

Comment:
```suggestion
- Introducing MongoDB scaler ([#1465](https://github.com/kedacore/keda/issues/1465))
```

Reply:
Yeah, I think it's better to call it `mongodb` scaler.

Reply:
We should keep that naming consistent, in the go code all the references to the scaler are as `mongo` not `mongodb`. Shouldn't we change that as well?

Reply:
You are right, i will change them to mongoDB,at the same time,add e2e tests for this scaler.

Reply:
Thanks!